src/main/java/net/javadiscord/javabot/systems/help/commands/CloseCommand.java

```diff
@@ -0,0 +1,22 @@
+package net.javadiscord.javabot.systems.help.commands;
+
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.Commands;
+
+/**
+ * A simple command that can be used inside reserved help channels to immediately unreserve them,
```

Comment:
I would change the Javadoc to (include) something like _An alias to `/unreserve`_ but that's more like an opinion.